docs/FAQ.md

```diff
@@ -0,0 +1,103 @@
+# Frequently Asked Questions
+
+## What is xdpw?
+
+xdpw is short for the name of this project, xdg-desktop-portal-wlr. This
+project seeks to add support for the screenshot, screencast, and possibly
+remote-desktop portal methods for wlroots based compositors.
+
+## What are xdg-desktop-portals?
+
+They were originally designed to offer access to system resources inside of
+flatpak sandboxes. More recently, large compositors have influenced many
+software makers to adopt these portals as standard mechanisms for screenshot
+and screencast functionalities in Wayland.
+
+## What is PipeWire?
+
+[PipeWire] s a server and user space API to deal with multimedia pipelines.
+It's scope is broad, seeking to provide a single mechanism for transporting
+video and audio streams to and from various applications in a unified fashion.
+
+For our purposes, it is the specified standard transport mechanism for the
+xdg-desktop-portal screencast API.
+
+## What version of PipeWire do I need?
+
+PipeWire 0.3 is the latest stable version of the API we use to provide video,
+so you will need to use a version of the PipeWire daemon greater than 0.3. Many
+applications still build against the PipeWire 0.2 libraries. The Pipewire 0.3
+daemon is backwards compatible with pipewire 0.2 consumers, but you may need to
+install the libraries from PipeWire 0.2 in addition to the full PipeWire 0.3
+package. Some distros that currently package PipeWire 0.3, provide compat
+packages (libpipewire02 in the case of Arch Linux), to support applications
+like Chrome and Firefox which are still built against PipeWire 0.2.
+
+## Will this let me share individual windows?
+
+No, only entire outputs. It would take significant work in wlroots, and likely
+a new protocol, to make this possible.
+
+## Will this let me share all of my outputs/displays at once?
+
+Not yet, no. It will likely never create a single stream that combines all of
+your outputs, but once we implement [better output selection], there is no
+reason xdpw cannot create multiple simultaneous streams of different outputs.
+
+## Will this let me share my screen in Zoom?
+
+For the web-based client, yes it will. For the native linux app, no, they use a
+Gnome specific API that this project does not seek to emulate.
+
+If this is something you care about, please leave [feedback for zoom], suggest
+an improvement for the Linux client, and ask them to support the
+xdg-desktop-portal screencast API instead of the proprietary Gnome one.
```

Comment:
proprietary sounds like its not open source... gnome specific / non standard is perhaps more fair?

Reply:
I'd prefer "private". I also think "proprietary" is fine in this context.

Reply:
https://www.thefreedictionary.com/proprietary
Yes. Non-standard is the correct word here.

Reply:
Yeah, I can see this both ways. I think I like private too.

Reply:
It's private, as in not intended for third-party use, reserved for internal GNOME mechanisms.